vendor/assets/javascripts/jquery.jnotify.js

```diff
@@ -160,7 +160,8 @@
 
 		// remove the notification
 		this.remove = function (callback){
-			var $msg = $note.find("." + options.classMessage), $parent = $msg.parent();
+
+			var $msg = $note.find("." + options.classMessage.replace(" ",",")), $parent = $msg.parent();
```

Comment:
Without this change you can't close dismissable alerts. Moving to pure bootstrap/patternfly alerts would fix it too, but it might be a big change. Maybe you can do that on a separate PR (before this), as you wish. I don't like changing the vendored js either.